src/main/service/GooglePlaceService.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  Client,
+  Place,
+  PlaceDetailsResponse,
+} from '@googlemaps/google-maps-services-js';
+import { PlaceField } from '../data/PlaceField';
+import { PlaceDetailsResponseData } from '@googlemaps/google-maps-services-js/dist/places/details';
```

Comment:
This is such a useful library!! 💯

Reply:
100% agree, having types is very useful and helps to detect errors at compilation. I would recommend we use the same library in the FE, except we only use it for its interfaces. This library isnt recommended for client side.

Reply:
Oh yeah we use `@types/googlemaps` on the FE which I think this library uses for types? I've used it a bit for the maps but I was thinking most of the Google Places types now would be abstracted by the types returned from the BE (which might be a good thing in terms of loosening the dependency between FE and Google Places)